docs/use/setup-wallet/index.md

```diff
@@ -36,19 +39,18 @@ This will allow you to send to an address of either format.
 
 
 ## Connect your wallet
-When using Centrifuge you can choose between pools deployed on Centrifuge Chain and on Ethereum. You can connect to both chains via our wallet component that allows you to pick the wallet of your choice. To do so, click on `Connect` in the top right corner. Then, first select the chain you want to connect with (currently Ethereum and Centrifuge):
-
-![](./images/wallet-component1.png#width=40%;)
+When using Centrifuge you can choose between pools deployed on Centrifuge Chain and on Ethereum. You can connect to both chains via our wallet component that allows you to pick the wallet of your choice. To do so, click on `Connect` in the top right corner. Then, first select the chain you want to connect with (currently Ethereum and Centrifuge). 
 
 In the second step select the wallet of your choice. On Centrifuge the system currently supports [Talisman](https://www.talisman.xyz/), [Subwallet](https://chrome.google.com/webstore/detail/subwallet-polkadot-extens/onhogfjeacnfoofkfgppdlbmlmnplgbn) and [Polkadot.js](https://polkadot.js.org/extension/).
 
```

Comment:
When using Centrifuge you can choose between pools deployed on Centrifuge Chain and on Ethereum. You can connect to both chains via our wallet component that allows you to pick the wallet of your choice. To do so, click on `Connect` in the top right corner. Then, first select the chain you want to connect with (currently Ethereum and Centrifuge). 

![image](https://github.com/centrifuge/documentation/assets/90972248/5fc4d25c-4517-4e17-9fe6-870300db3b44)

In the second step select the wallet of your choice. On Centrifuge Chain the system currently supports [Talisman](https://www.talisman.xyz/), [Subwallet](https://www.subwallet.app/download.html?lang=1), [Polkadot.js](https://polkadot.js.org/extension/), [Fearless Wallet](https://fearlesswallet.io/) and EVM compatible wallets [MetaMask](https://metamask.io/download.html), [Coinbase Wallet](https://www.coinbase.com/wallet) and multiple other wallets through [Wallet Connect](https://walletconnect.com/).